plugins/view-resources/src/components/filter/FilterTypePopup.svelte

```diff
@@ -85,6 +85,18 @@
     mixin: ClassFilters
   ): void {
     const ignoreKeys = new Set(mixin.ignoreKeys ?? [])
+
+    if (mixin.strict) {
```

Comment:
When is strict we should don't try build keys from hierarchy (desc and asc mixins)

Reply:
Done.